frontend/src/api/client.ts

```
@@ -32,87 +33,101 @@ function mockUrl(path: string): string | null {
-  const tryMock = async (): Promise<T> => {
-    const url = mockUrl(path);
-    if (!url) {
-      throw new Error(`No mock mapped for ${path}`);
+  let res: Response;
+  try {
+    // Если forceMock — не трогаем сеть
+    if (forceMock && mockUrl) {
+      res = await useMock();
+    } else {
+      // Сначала — основной эндпойнт
+      res = await fetch(primary, {
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)